### PR TITLE
Update docs for 1.13.0 release

### PR DESCRIPTION
**Pull request by jandubois**
* Added version information for bundled utilities
* Added WASM tutorial and WASM info to Containers > General page

Fixes https://github.com/rancher-sandbox/rancher-desktop/issues/6554

### CODE REVIEW

File: docs/tutorials/working-with-webassembly.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+Rancher Desktop 1.13.0 added experimental support for running WebAssembly (WASM) applications. This feature needs to be enabled in  [Preferences > Container Engine > General](../ui/preferences/container-engine/general.md).
+
+:::caution warning
+Note that when using the `moby` container engine, enabling the WASM feature switches to a different image store, so previously built or downloaded images will not be available and must be built or downloaded again. The images are not lost; Rancher Desktop will switch back to the old image store when WASM is disabled again.
+:::
+
+## Managing containerd WASM shims
+
+To run WebAssembly applications on a container runtime requires a "shim" for each WASM runtime/framework that is going to be used.
```
Comment (gunamata) [inline docs/tutorials/working-with-webassembly.md:17]:
```suggestion
Running WebAssembly applications on a container runtime requires a specific containerd "shim" corresponding to the WASM runtime/framework used. There are various providers such as spin, runwasi, slight etc who provide containerd shims.
```

File: docs/tutorials/working-with-webassembly.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+Rancher Desktop 1.13 comes bundled with a the `containerd-spin-shim-v2` shim preinstalled. Future releases are expected to download additional shims automatically when the feature is enabled.
```
Comment (gunamata) [inline docs/tutorials/working-with-webassembly.md:19]:
```suggestion
Rancher Desktop 1.13 comes bundled with the `containerd-spin-shim-v2` shim preinstalled. Future releases are expected to download additional shims automatically when the feature is enabled.
```

File: docs/tutorials/working-with-webassembly.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+For now additional shims can be installed by the user into the `containerd-shims` cache directory on the host. The location is
```
Comment (gunamata) [inline docs/tutorials/working-with-webassembly.md:21]:
```suggestion
However, you can install additional shims by manually downloading and copying the shim binaries to the `containerd-shims` cache directory on the host. The location is
```

**Review by gunamata — CHANGES_REQUESTED** [review]
Just a couple of minor suggestions. It looks good to me otherwise. Thank you.

**Comment by gunamata** [comment]
I have just noticed the CI build failed. Some issue with the link to the WASM tutorial page from the Preferences page.

File: docs/tutorials/working-with-webassembly.md (hunk 1 above)
Comment (jandubois) [reply docs/tutorials/working-with-webassembly.md:17]: I've applied the first part of the suggestion. But the last sentence is not really helpful: either the user knows about this already (and therefore doesn't need the info), or they don't, in which case it doesn't tell them anything about the runtimes, the shims, where to get them etc.

Let's expand on this for the next release to have more beginner-friendly docs.

File: docs/tutorials/working-with-webassembly.md (hunk 2 above)
Comment (jandubois) [reply docs/tutorials/working-with-webassembly.md:19]: Changed

File: docs/tutorials/working-with-webassembly.md (hunk 3 above)
Comment (jandubois) [reply docs/tutorials/working-with-webassembly.md:21]: The whole document is written in passive voice (which I found a bit painful), so adding a "you can do this" sentence feels out of place. If we want to change the tone, then the whole page should be adjusted accordingly.

**Review by gunamata — APPROVED** [review]

**Pull request merged**